README.md

```diff
@@ -56,7 +56,7 @@ You need to install Node.js and then the development tools. Node.js comes with a
 Either clone this repository or fork it on GitHub and clone your fork.
 
 ```
-git clone https://github.com/congarevenuecloud/e-commerce
+git clone https://github.com/congarevenuecloud/partner-commerce
```

Comment:
Why did we update the template to partner-commerce ? Shouldn't it be e-commerce itself?